CMakeLists.txt

```diff
@@ -551,8 +551,8 @@ if (WIN32)
     "${DDK_ROOT}/tools/tracing/${DDK_SFX}/dbghelp.dll"
     "${PROGFILES}/Microsoft Visual Studio */Common7/IDE/Remote Debugger/${ARCH_SFX}/dbghelp.dll"
     "${PROGFILES32}/Windows Kits/*/Debuggers/${ARCH_SFX}/dbghelp.dll"
-    # For Visual Studio 2010+, x64 dbghelp.dll resides in Program
-    # Files (x86) rather than Program Files.
+    # In case if SDK is not installed and we have Visual Studio, dbghelp.dll may be found in the
+    # Visual Studio's directory in subfolder Remote Debugger/x86 or /x64 folders (xref i#1956).
```

Comment:
I think I don't understand.
Do you mean that VS 2005 and 2008 will put dbghelp.dll in C:\Program Files (in case of x64 machine) and VS 2010+ will put dbghelp.dll in C:\Program Files (x86). Right ?